app/client/views/homes/assignManger/schema.js

```diff
@@ -0,0 +1,25 @@
+import SimpleSchema from "simpl-schema";
+SimpleSchema.extendOptions(["autoform"]);
+
+const assignManagerSchema = function(users, groupId) {
+  return new SimpleSchema({
+    groupId: {
+      type: String,
+      optional: false,
+      autoform: {
+        value: groupId,
+        type: "hidden",
+        label: false
+      }
+    },
+    userId: {
+      type: String,
+      optional: false,
+      autoform: {
+        options: users
```

Comment:
I believe you should be able to just put the `getAssignManagerSchemaUsers()` function here:

```js
function getAssignManagerSchemaUsers () {
 const users = Meteor.users
      .find()
      .fetch()
      .map(user => {
        let address = "Unknown";
        if (user.emails.length > 0) {
          address = user.emails[0].address || "Unknown";
        }
        return {
          label: address,
          value: user._id
        };
      });

  return users;
```

Reply:
I.e. try to keep things as close to where they are used as possible.